lib/config.js

```diff
@@ -3,17 +3,18 @@ module.exports = {
     NODE_ENV: process.env.NODE_ENV,
     LOG_DIR: process.env.LOG_DIR || './logs',
     SYSTEM_CODE: process.env.SYSTEM_CODE,
+    KMT_SECRET: process.env.KMT_SECRET,
+    DOMAIN: process.env.DOMAIN,
+    BASE_PATH_URL: process.env.BASE_PATH_URL,
     API_GATEWAYKEY:  process.env.API_GATEWAYKEY,
     API_GATEWAY_HOST: process.env.API_GATEWAY_HOST,
-    LICENCE_DATA_SVC_HOST:process.env.LICENCE_DATA_SVC_HOST,
-    ACC_LICENCE_SVC_HOST:process.env.ACC_LICENCE_SVC_HOST,
     MEMCOM_APIKEY:process.env.MEMCOM_APIKEY,
     API_AUTH_CLIENT_ID:process.env.API_AUTH_CLIENT_ID,
     LOGIN_URL: process.env.LOGIN_URL,
     LOGOUT_URL: process.env.LOGOUT_URL,
+    LICENCE_DATA_SVC_HOST:process.env.LICENCE_DATA_SVC_HOST,
     ACC_LICENCE_SVC_HOST:process.env.ACC_LICENCE_SVC_HOST,
-    KMT_SECRET: process.env.KMT_SECRET,
-    DOMAIN: process.env.NODE_ENV === 'development' ? 'local.ft.com' : '.ft.com'
+    ACQ_CTX_SVC_HOST:process.env.ACQ_CTX_SVC_HOST,
   },
 
   initialise: function (config) {
```

Comment:
Do I still need to run `kmtUtils.initialise(require('../utilities.Config'));` if it doesn't really do anything?
From what I see, it checks if the `config` object has keys. It obviously does and so my config is ignored.